package.xml

```diff
@@ -1,7 +1,7 @@
 <package format="2">
-  <name>qt_gui_icons</name>
+  <name>tango_icons_vendor</name>
   <version>0.0.0</version>
-  <description>qt_gui_icons provides the public domain Tango icons for Windows (<a href="http://tango.freedesktop.org/Tango_Icon_Library">Tango Icon Library</a>) from the <a href="http://tango.freedesktop.org/Tango_Desktop_Project">Tango Desktop Project</a></description>
+  <description>tango_icons_vendor provides the public domain Tango icons for non-linux systems (<a href="http://tango.freedesktop.org/Tango_Icon_Library">Tango Icon Library</a>) from the <a href="http://tango.freedesktop.org/Tango_Desktop_Project">Tango Desktop Project</a></description>
```

Comment:
Shouldn't there be a dependency on [tango-icon-theme](https://github.com/ros-visualization/qt_gui_core/blob/57bb428246a269f755f4d5a8b31dabcbe0b78d93/qt_gui/package.xml#L32) to ensure the icons are available on Linux?

Reply:
Qt installs its own icons. 

>  - Note: By default, only X11 will support themed icons. In order to use themed icons on Mac and Windows, you will have to bundle a compliant theme in one of your themeSearchPaths() and set the appropriate themeName().

Reply:
Can you please clarify how your comment answers my question. If you think it shouldn't be installed can you argue why removing `tango-icon-theme` doesn't have any effect on Linux for rqt?

Reply:
the problem with icons appears only on Windows and MacOS. By the way this dependency does not make sense because rosdep it's only working on Linux. This will not fix the issue on MacOS and Windows. In the PR that I opened to modify the logic I have changed this dependecy for tango_icons_vendor https://github.com/ros-visualization/qt_gui_core/pull/222 (it's not updated yet but you can see that the name correspond to the previous name).

Reply:
And that patch *removes* the dependency on `tango-icon-theme`. The rosdep key doesn't have a mapping for macOS but it could (see https://github.com/ros/rosdistro/blob/341974cc4cd5842efdca76993ae4fbf67f50b1a0/rosdep/base.yaml#L5677-L5690).

So the question I have and repeat: how does removing the dependency on Linux on `tango-icon-theme` affect rqt? I assume that until know we are relying on these icons and are actually using them.

Reply:
I don't have tango-icon-theme package installed on my computer and I'm able to see the icons on Linux

Reply:
Looking into my configuration I have installed in system Adwaita which is the default GNOME theme.  My proposal is:

 - Force to install tango icons in all the platforms and remove the tango-icon-theme dependency. (Icons will look the same in all the platforms)

Reply:
> Force to install tango icons in all the platforms and remove the tango-icon-theme dependency.

Since there is an upstream Debian package for it we should not provide out own on that platform.

Please answer my previous question: how does removing the dependency on Linux on `tango-icon-theme` affect rqt?

* If the answer is "it doesn't at all - we are not using Tango icons in any case on Linux" then the dependency can just be dropped.
* If the answer is "it changes the UI in some cases since we use the Tango icon in some cases on Linux" then the dependency should be added to this repo (and removed from `qt_gui` which will be updated to depend on this package instead).

Reply:
I have tested this on Gnome Ubuntu, I don't know how other Linux distribution handle this. Probably all of them that use Gnome do the same as Ubuntu.

By default Ubuntu set the icon theme to `ubuntu-mono-light`. I have installed on top `tango-icon-theme` but the default theme icon in my system remains the same. I need to execute the following command to set the Tango icon available:

`gsettings set org.gnome.desktop.interface icon-theme 'Tango'`

What I'm trying to say is that you should have an icon theme available in a Linux distribution even if tango icon theme is not installed.

The answer to this question in the first one:

> If the answer is "it doesn't at all - we are not using Tango icons in any case on Linux" then the dependency can just be dropped.

But I think my proposal is still valid:

> - Force to install tango icons in all the platforms and remove the tango-icon-theme dependency. (Icons will look the same in all the platforms)

Reply:
There are some icon themes that you may have configured in your system which are not going to work because of the available size or the icon. That's why I think we should use the tango_icons_vendor in all platforms

Reply:
I don't think we should override a working theme on Linux. The idea of desktop themes is to provide a consistent look between application.